edx_proctoring/templates/proctoring/seq_timed_exam_footer.html

```diff
@@ -0,0 +1,11 @@
+{% load i18n %}
+<div class="footer-sequence">
+  <h4> {% trans "What if i need more time " %}? </h4>
+    <p>{% blocktrans %}
+        {{platform_name}} ocassionally grants more time for students with disabilities and difficult conditions.
```

Comment:
Wouldn't this be up to the course team, not the platform?
"If you have disabilities or are taking the exam in difficult conditions, you might be eligible for additional time allowance on timed exams. Ask your instructor or course staff for information about additional time allowances."